src/components/ReportActionItem/TaskView.js

```diff
@@ -0,0 +1,148 @@
+import React, {useEffect} from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import reportPropTypes from '../../pages/reportPropTypes';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import withWindowDimensions from '../withWindowDimensions';
+import withCurrentUserPersonalDetails, {withCurrentUserPersonalDetailsPropTypes} from '../withCurrentUserPersonalDetails';
+import compose from '../../libs/compose';
+import Navigation from '../../libs/Navigation/Navigation';
+import ROUTES from '../../ROUTES';
+import MenuItemWithTopDescription from '../MenuItemWithTopDescription';
+import MenuItem from '../MenuItem';
+import styles from '../../styles/styles';
+import * as ReportUtils from '../../libs/ReportUtils';
+import * as PersonalDetailsUtils from '../../libs/PersonalDetailsUtils';
+import * as UserUtils from '../../libs/UserUtils';
+import * as StyleUtils from '../../styles/StyleUtils';
+import * as Task from '../../libs/actions/Task';
+import CONST from '../../CONST';
+import Checkbox from '../Checkbox';
+import convertToLTR from '../../libs/convertToLTR';
+import Text from '../Text';
+import Icon from '../Icon';
+import getButtonState from '../../libs/getButtonState';
+import PressableWithSecondaryInteraction from '../PressableWithSecondaryInteraction';
+import * as Session from '../../libs/actions/Session';
+import * as Expensicons from '../Icon/Expensicons';
+
+const propTypes = {
+    /** The report currently being looked at */
+    report: reportPropTypes.isRequired,
+
+    /** Whether we should display the horizontal rule below the component */
+    shouldShowHorizontalRule: PropTypes.bool.isRequired,
+
+    ...withLocalizePropTypes,
+
+    ...withCurrentUserPersonalDetailsPropTypes,
+};
+
+function TaskView(props) {
+    useEffect(() => {
+        Task.setTaskReport({...props.report, isExistingTaskReport: true});
+    }, [props.report]);
+
+    const taskTitle = convertToLTR(props.report.reportName || '');
+    const isCompleted = ReportUtils.isCompletedTaskReport(props.report);
+    const isOpen = ReportUtils.isOpenTaskReport(props.report);
+    const isCanceled = ReportUtils.isCanceledTaskReport(props.report);
+
+    return (
+        <View>
+            <PressableWithSecondaryInteraction
```

Comment:
What was reason for using `PressableWithSecondaryInteraction` in `TaskView` instead of just `PressableWithFeedback`?
I don't see any secondary interaction.